index.bs

```diff
@@ -1395,6 +1396,20 @@ The {{WebTransportSendStream}}'s [=transfer steps=] and
            1. [=Resolve=] |p| with |stats|.
      1. Return |p|.
 
+: <dfn for="WebTransportSendStream" method>getAtomicWriter()</dfn>
+:: For transactional applications where blocking on flow control can be an
+   issue, the following alternative writer API gives the application
```

Comment:
A citation for https://datatracker.ietf.org/doc/html/rfc9204#section-2.1.3 might help here.  It's not a great citation, but it highlights the sort of issue this might be avoiding.

Reply:
Lucas reminded me of [RFC 9308](https://www.rfc-editor.org/rfc/rfc9308.html#section-4.4) which has a lot more and more useful text.